src/zenml/integrations/sklearn/__init__.py

```diff
@@ -20,7 +20,7 @@ class SklearnIntegration(Integration):
     """Definition of sklearn integration for ZenML."""
 
     NAME = SKLEARN
-    REQUIREMENTS = ["sklearn==0.24.2"]
+    REQUIREMENTS = ["scikit-learn==1.0.1"]
```

Comment:
Shouldnt this be >=1.0.1? Lets make the requirements looser

Reply:
Agreed, I'll remove the version requirement completely for now, similar to the other integrations (I'll also add airflow==2.2.0, as in this case it actually seems to be only working with that version)